src/Microsoft.DocAsCode.App/RunServe.cs

```diff
@@ -43,11 +47,104 @@ public static void Exec(string folder, string host, int? port)
             Console.WriteLine($"Serving \"{folder}\" on {url}. Press Ctrl+C to shut down.");
             using var app = builder.Build();
             app.UseFileServer(fileServerOptions);
-            app.Run();
+
+            if (openBrowser)
+            {
+                string relativePath = openBrowserRelativePath;
+                var launchUrl = string.IsNullOrEmpty(relativePath)
+                    ? url
+                    : Path.Combine(url, ResolveOutputHtmlRelativePath(folder, relativePath));
+
+                // Start web server.
+                app.Start();
+
+                // Launch browser process.
+
+                Console.WriteLine($"Launching browser with url: {url}.");
```

Comment:
`launchUrl` or otherwise the URL is already printed in line 47:
```suggestion
                Console.WriteLine($"Launching browser with url: {launchUrl}.");
```

Reply:
As you pointed out. I’ve fixed at latest commit.